### PR TITLE
Initial packaging

### PR DESCRIPTION
**Pull request by rominf**
I added license (is GPL v3 okay?) and packaging files. Now anyone can install your timeit_plot via pip:

```
sudo pip install https://github.com/rominf/timeit_plot/archive/master.zip
```

It's my first packaged python script, beware of bugs.

I also added support for python 3 via __future__ and zip() wrapping.

### CODE REVIEW

**Comment by cseager** [comment]
Oh hello! I just saw this, sorry I've been away from github for a while. I will take a look... I have never made a pip package before either. I've also made a couple of small changes in the last few months to make the range selection more straightforward, but I haven't gotten around to cleaning/testing/pushing it just yet.

File: timeit_plot.py [hunk 1]
```diff
@@ -12,22 +13,22 @@
 
 def timeit_compare(funcs, inputs, setups='pass', **kwargs):
     """Compares speed of functions across input conditions.
-
```
Comment (cseager) [inline timeit_plot.py:15]: is there a diff here? the two docstrings look the same to me.

File: timeit_plot.py [hunk 2]
```diff
@@ -197,10 +198,10 @@ def timeit_heatmap(data, xlabel='xlabel', ylabel='ylabel', **kwargs):
 
 def substitute_titles(label, series):
     ordered_axes=["x", "y", "z"]
-    try:
```
Comment (cseager) [inline timeit_plot.py:200]: Is there just some whitespace difference I'm not seeing here? like changing tabs to spaces maybe?

**Comment by rominf** [comment]
I edited this file with vim (+python-mode enabled). It removed trailing whitespace in all lines when I saved file.
Sorry, now I'm busy — I have to prepare for exam of probability theory. I've done some other changes that I forgot to push, but I need to remember what have I done and review it myself. I think you can push your changes without accepting this commit — I'll rebase in several days and push it again.

**Pull request closed without merging**